pkg/services/notifications/email.go

```diff
@@ -7,6 +7,7 @@ import (
 
 type Message struct {
 	To           []string
+	SingleEmail  bool
```

Comment:
Would like to change this so that single email is the default, currently this makes single email default to false. Suggests renaming this to `MultipleEmails`.